avocado_vt/conf.d/vt.conf

```diff
@@ -4,7 +4,11 @@ backup_image_before_test = True
 # Restore image after testing (if backup present)
 restore_image_after_test = True
 # Keep guest running between tests (faster, but unsafe)
-keep_guest_running=False
+keep_guest_running = False
+# Make the temporary dir path persistent across jobs if needed.
+# By default the data in the temporary directory will be wiped oafter each test
+# in some cases and after each job in others.
+persistent_tmp_dir =
```

Comment:
And frankly, I do prefer the name to be `tmp_dir` instead, because VT doesn't care of whether the given directory is truly persistent or not. From VT's point of view it just makes the path of `tmp_dir` to be user customized, otherwise, nothing changed.

Reply:
Alright, will do in the next push